pom.xml

```diff
@@ -335,13 +335,19 @@
 
         <dependency>
             <groupId>org.eclipse.jetty.websocket</groupId>
-            <artifactId>websocket-jetty-api</artifactId>
+            <artifactId>jetty-websocket-jetty-api</artifactId>
             <version>${jetty.version}</version>
         </dependency>
 
         <dependency>
             <groupId>org.eclipse.jetty.websocket</groupId>
-            <artifactId>websocket-jetty-server</artifactId>
+            <artifactId>jetty-websocket-jetty-server</artifactId>
+            <version>${jetty.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.eclipse.jetty.ee10.websocket</groupId>
```

Comment:
Just been doing some reading - if we change this to ee9 it should behave the same way as the previous version?

Reply:
Theoretically, ee9 with Jakarta 5 should be a valid set of dependencies to go with Jetty 12, but I think we tried that before after first noticing the issues with the websocket and also got (a different set of) errors with that.
Might be worth revisiting given we're stalled out on the ee10 version though...